### PR TITLE
CB 16620  Lazy init exception occurred on rds config listing

### PR DESCRIPTION
**Pull request by topolyai5**
See detailed description in the commit message.

### CODE REVIEW

**Review by lnardai — APPROVED** [review]

**Review by horadla23 — COMMENTED** [review]
LGTM, although it would be good if you could test this before merging it

**Comment by lnardai** [comment]
Since it's targeted as a hotfix, please make sure to validate the change locally. Team does not wish to merge it until this is done.

**Comment by topolyai5** [comment]
I have successfully tested on my machine:
image

**Pull request merged**
